src/safeds/ml/classical/classification/_ada_boost.py

```diff
@@ -13,27 +13,34 @@
 
 
 class AdaBoost(Classifier):
-    """Ada Boost classification.
+    """
+    Ada Boost classification.
 
     Parameters
     ----------
     learning_rate : float
         Weight applied to each classifier at each boosting iteration. A higher learning rate increases the contribution
         of each classifier. Has to be greater than 0.
+    maximum_number_of_learners: int
+        The maximum number of learners at which boosting is terminated. In case of perfect fit, the learning procedure
+        is stopped early. Has to be greater than 0.
 
     Raises
     ------
     ValueError
-        If the learning rate is less than or equal to 0.
+        If the learning rate or maximum_number_of_learners are less than or equal to 0
```

Comment:
We should consistently write either the meaning of a parameter (learning rate) or its name (maximum_number_of_learners). I'd vote for the latter since it's more obvious. Might also put the names of the parameters in backticks.